app/utils.py

```diff
@@ -524,6 +529,22 @@ def redact_mobile_number(mobile_number, spacing=""):
     return "".join(mobile_number_list)
 
 
+def get_letter_printing_statement(status, created_at):
+    created_at_dt = parser.parse(created_at).replace(tzinfo=None)
+    if letter_can_be_cancelled(status, created_at_dt):
+        return 'Printing starts {} at 5:30pm'.format(printing_today_or_tomorrow())
+    else:
+        printed_datetime = utc_string_to_aware_gmt_datetime(created_at) + timedelta(hours=6, minutes=30)
+        if printed_datetime.date() == datetime.now().date():
+            return 'Printed today at 5:30pm'
+        elif printed_datetime.date() == datetime.now().date() - timedelta(days=1):
+            return 'Printed yesterday at 5:30pm'
+
+        printed_date = printed_datetime.strftime('%d %B').lstrip('0')
+
+        return 'Printed on {} at 5:30pm'.format(printed_date)
```

Comment:
I think this is more consistent with the message that says _Printing starts tomorrow at 5:30pm_:
```suggestion
        return 'Printing started at 5:30pm on {}'.format(printed_date)
```

Reply:
Ignore that, just read the code more carefully and can see why you’ve done what you done